feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/nominationPools/selecting/SearchNominationPoolInteractor.kt

```diff
@@ -48,7 +43,6 @@ class SearchNominationPoolInteractor(
                 val address = it.address(stakingOption.chain)
                 name?.contains(query).orFalse() || address.startsWith(query) || it.hasId(query)
             }
-                .sortedWith(comparator)
```

Comment:
Sorting was actually quite reasonable I think. In case we found multiple matching pools it is great to sort them